### PR TITLE
CFE-3432: Acceptance test testing comment-attribute with custom promise types

### PR DESCRIPTION
**Pull request by larsewi**

### CODE REVIEW

**Comment by larsewi** [comment]
@cf-bottom Jenkins please :)

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7481/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7481/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7481/

**Review by craigcomstock — APPROVED** [review]

File: tests/acceptance/30_custom_promise_types/noop_promises.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+from cfengine import PromiseModule, ValidationError, Result
+
+
+class AppendPromiseTypeModule(PromiseModule):
+    def __init__(self):
+        super().__init__("noop_promise_module", "0.0.1")
+
+    def validate_promise(self, promiser, attributes):
+        if type(promiser) != str:
+            raise ValidationError("Promiser must be of type string")
```
Comment (vpodzime) [inline tests/acceptance/30_custom_promise_types/noop_promises.py:31]: I believe it's guaranteed to be a string.

**Review by vpodzime — APPROVED** [review]
ACK otherwise.

**Comment by larsewi** [comment]
@cf-bottom Jenkins please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7493/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7493/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7493/

**Comment by larsewi** [comment]
@cf-bottom Jenkins please

**Review by vpodzime — APPROVED** [review]
ACK

**Comment by larsewi** [comment]
@cf-bottom Jenkins please :)

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7514/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7514/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7514/

**Review by olehermanse — APPROVED** [review]

**Comment by larsewi** [comment]
@cf-bottom Jenkins please :)

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7519/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7519/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7519/

**Comment by larsewi** [comment]
@cf-bottom Jenkins please :)

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7528/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7528/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7528/

**Comment by larsewi** [comment]
@cf-bottom Jenkins please :)

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7530/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7530/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7530/

**Pull request merged**